core/src/main/java/com/sequenceiq/cloudbreak/reactor/api/event/cluster/ClusterDbCertRotationResult.java

```diff
@@ -0,0 +1,21 @@
+package com.sequenceiq.cloudbreak.reactor.api.event.cluster;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.sequenceiq.cloudbreak.reactor.api.ClusterPlatformResult;
+
+public class ClusterDbCertRotationResult extends ClusterPlatformResult<ClusterDbCertRotationRequest> {
```

Comment:
Why not subclass `StackEvent` instead?

Reply:
Let us leave it as is for now; can be investigated & refactored later if needed.